rnd/autogpt_server/autogpt_server/util/service.py

```diff
@@ -43,7 +46,9 @@ def run(self):
 
 class AppService(AppProcess):
     shared_event_loop: asyncio.AbstractEventLoop
+    event_queue: AsyncEventQueue = AsyncRedisEventQueue()
     use_db: bool = True
+    use_redis: bool = True
```

Comment:
Nit: It seems like most services don't need Redis, so maybe make this default `False` and enable it on the services that do need it.